hammerblade/torch/riscv.mk

```diff
@@ -96,13 +94,8 @@ kernel.riscv: $(KERNEL_OBJS)
 	$(RISCV_LINK) $(KERNEL_OBJS) $(SPMD_COMMON_OBJECTS) \
 		-L. -l:$(BSG_MANYCORE_LIB) -o $@ $(filter-out -nostdlib,$(RISCV_LINK_OPTS))
 
-$(KERNEL_DIS): %.dis: %.o
-	$(RISCV_BIN_DIR)/riscv32-unknown-elf-dramfs-objdump -M numeric --disassemble-all -S $< > $@
-
-kernel.dis: kernel.riscv
-	$(RISCV_BIN_DIR)/riscv32-unknown-elf-dramfs-objdump -M numeric --disassemble-all -S $< > $@
-
-all.dis: kernel.dis $(KERNEL_DIS)
+%.dis: %.o
```

Comment:
I think in your local branch, you reverted the commit that leads to empty /torch/riscv/. So you would need to "revert the revert"

Reply:
Oh yes, there was a problem with merge which I'd fixed but not pushed; done that now